src/AutoMerge/public.jl

```diff
@@ -1,5 +1,64 @@
 """
-    run(env, cicfg::CIService; kwargs...)
+    run([env, cicfg::CIService]; kwargs...)
+
+Run the `RegistryCI.AutoMerge` service.
+
+# Arguments
+
+- `env`:
+- `ciccfg`:
```

Comment:
not sure how to describe this... @DilumAluthge any ideas?

Reply:
Which one, `env` or `ciccfg`?

Reply:
`ciccfg`

Reply:
Hmmm.

"Configuration struct describing the continuous integration (CI) environment in which AutoMerge is being run"?

Reply:
Yeah, that makes sense. I think then those structs maybe should be documented too, but that shouldn't hold up this PR

Reply:
Done, take a look